stdlib/builtins.pyi

```diff
@@ -221,8 +221,12 @@ class int:
     def __rmod__(self, __x: int) -> int: ...
     def __rdivmod__(self, __x: int) -> tuple[int, int]: ...
     @overload
+    def __pow__(self, __x: int, __modulo: Literal[0]) -> NoReturn: ...
+    @overload
     def __pow__(self, __x: Literal[2], __modulo: int | None = ...) -> int: ...
     @overload
+    def __pow__(self, __x: Literal[3], __modulo: int | None = ...) -> int: ...
+    @overload
```

Comment:
These can be combined with `Literal[2, 3]`. The original motivation for including only `2` was that it is used way more often than other powers.